packages/core/src/columns/columnType.ts

```diff
@@ -189,6 +189,45 @@ export const pushColumnData = <
   );
 };
 
+// types to be assigned to the column with .asType
+type ColumnTypeTypes<
```

Comment:
`ColumnDataTypes`: types of column data. Types of column type don't seem right.

```suggestion
type ColumnDataTypes<
```